### PR TITLE
Suppress everest logger output to the console

### LINKED ISSUE #9269: DEBUG and INFO logging bleeding to console, should be only added to the log files
How to reproduce:

`everest run test-data/everest/math_func/config_minimal.yml`

Example:

```
INFO:everest:everest:'11.1.0b7.dev72+gc4a2062e5.d20241118'
ropt:'0.9.0'
ert:'11.1.0b7.dev72+gc4a2062e5.d20241118'
INFO:everest:Output directory: .../ert/test-data/everest/math_func/everest_output
DEBUG:everest:Namespace(config_file='.../ert/test-data/everest/math_func/config_minimal.yml', debug=False)
 * Serving Flask app 'everest.detached.jobs.everserver'
 * Debug mode: off
INFO:werkzeug:WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on all addresses (0.0.0.0)
 * Running on https://127.0.0.1:5000
 * Running on https://192.168.1.241:5000
INFO:werkzeug:Press CTRL+C to quit
INFO:everest:Using random seed: 123
INFO:everest:To deterministically reproduce this experiment, add the above random seed to your configuration file.
DEBUG:everest:Storing internal data in .../ert/test-data/everest/math_func/everest_output/.internal_data
DEBUG:everest:Storing internal data in .../ert/test-data/everest/math_func/everest_output/.internal_data
DEBUG:everest:Creating job description .../ert/test-data/everest/math_func/everest_output/.jobs
INFO:everest:Using random seed: 123
INFO:everest:To deterministically reproduce this experiment, add the above random seed to your configuration file.

```

### PR DESCRIPTION
**Pull request by verveerpj**
**Issue**
Resolves #9269

**Approach**
Redo logging setup using dict configuration, which mysteriously fixes the issue.

- [ ] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Added appropriate release note label
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Make sure unit tests pass locally after every commit (`git rebase -i main
      --exec 'pytest tests/ert/unit_tests -n logical -m "not integration_test"'`)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

**Comment by verveerpj** [comment]
Unfortunately, it has proven difficult to write a test for this, because `pytest` interferes with the logging. It is possible to manipulate that with the `caplog` fixture, but that seemed to not work nicely with the `capsys` fixture, which is needed to check the output for console.

Therefore I will not write a test. If we experience a regression with this issue, it will become evident immediately since it will show up in the console.

File: src/everest/detached/jobs/everserver.py [hunk 1]
```diff
@@ -193,41 +197,51 @@ def _write_hostfile(host_file_path, host, port, cert, auth) -> None:
...
-        log_level=logging.INFO,
-    )
-
-    configure_logger(
-        name="everserver",
-        file_path=os.path.join(detached_node_dir, "endpoint.log"),
-        log_level=logging.INFO,
-    )
+def _configure_loggers(config: EverestConfig) -> None:
+    def _make_handler(path: Path, log_level: str | int = "INFO") -> dict[str, Any]:
```
Comment (DanSava) [inline src/everest/detached/jobs/everserver.py:201]: Nitpicky comment: I would rename this to `make_handler_config`

File: src/everest/detached/jobs/everserver.py [hunk 2]
```diff
@@ -193,41 +197,51 @@ def _write_hostfile(host_file_path, host, port, cert, auth) -> None:
...
 
-    configure_logger(
-        name=EVEREST,
-        file_path=os.path.join(everest_logs_dir, "everest.log"),
-        log_level=logging_level,
-        log_to_azure=True,
-    )
+    detached_dir = Path(ServerConfig.get_detached_node_dir(config.output_dir))
+    log_dir = Path(config.log_dir)
+    logging_dict = {
```
Comment (DanSava) [inline src/everest/detached/jobs/everserver.py:212]: Would you consider naming this `logging_config`?

File: src/everest/detached/jobs/everserver.py [hunk 3]
```diff
@@ -159,6 +162,7 @@ def get_opt_progress(
         ssl_certfile=server_config["cert_path"],
         ssl_version=ssl.PROTOCOL_SSLv23,
         ssl_keyfile_password=server_config["key_passwd"],
+        log_level=logging.CRITICAL,
```
Comment (DanSava) [inline src/everest/detached/jobs/everserver.py:165]: :heart: 

The messages from the HTTP server were driving me crazy during debugging

File: src/everest/detached/jobs/everserver.py [hunk 4]
```diff
@@ -193,41 +197,51 @@ def _write_hostfile(host_file_path, host, port, cert, auth) -> None:
...
-        file_path=os.path.join(detached_node_dir, "simulations.log"),
-        log_level=logging.INFO,
-    )
-
-    configure_logger(
-        name="everserver",
-        file_path=os.path.join(detached_node_dir, "endpoint.log"),
-        log_level=logging.INFO,
-    )
+def _configure_loggers(config: EverestConfig) -> None:
```
Comment (DanSava) [inline src/everest/detached/jobs/everserver.py:200]: Is it possible for this function to only get the required components from EverestConfig and not the entire object?

**Review by DanSava — APPROVED** [review]
Thank you :heart:

**Pull request merged**